src/app/api/v1/admin/feature-flags/[flagId]/route.ts

```diff
@@ -0,0 +1,33 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { NextRequest, NextResponse } from "next/server";
+import { getToken } from "next-auth/jwt";
+import {
+  getFeatureFlagByName
+} from "../../../../../../db/tables/featureFlags";
+import { isAdmin } from "../../../../utils/auth";
+import appConstants from "../../../../../../appConstants";
+
+//  import appConstants from "../../../../../appConstants";
+
+export async function GET(req: NextRequest) {
+  const token = await getToken({ req });
+  if (typeof token?.email === "string" && isAdmin(token?.email)) {
```

Comment:
Looks like we're doing this pattern in at least 4&times;.
Should we move the `typeof token?.email === "string"` check into the `isAdmin(...)` method?

Reply:
I think it's redundant, simplified